### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by DamenaM**
### PR   add a contact form and finish the mobile version

- [ ] add a contact form
- [ ] data validation email 
- [ ] validation Name field accepts up to 30 chars.
- [ ] validation  Email field accepts only emails.
- [ ] fix linter error

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -155,43 +156,70 @@ <h3 class="Multi-Post_05" >Multi-Post Stories </h3>
...
+            </div>
+            <div class="form-group">          
+                <input class="texform"  type="email"  id="email"  name="email"   placeholder="Enter Your Email"  required  />
+            </div>
+            <div class="form-group2">
+                <input class="texform2"  name="comment"  id="comment" maxlength="500"s  placeholder="Write your message here..."  required  />
+            </div>
+            <div class="buttonsee2">
+                <button class="contactbutton"> Get It Touch </button>
+            </div>
```
Comment (mckent05) [inline index.html]: **OPTIONAL** - For semantic reasons, I suggest that yu do not need to wrap all your form input in a `div` tag, please let them be directly under the `form` tag. Please fix this 🙏

File: index.html [hunk 2]
```diff
@@ -155,43 +156,70 @@ <h3 class="Multi-Post_05" >Multi-Post Stories </h3>
...
 </section>
+<section class="cardcontact">
+    <form class="cardcontact1" action="https://formspree.io/f/mwkzayzq" method="post" >
+        
+        <div class="sectioncontact">
+            <h2  class="contactme" >Contact me </h2>
+        <p class="contact"> If you have an application you are interested 
+            in developing, a feature that you need built or a project that 
+            needs coding. I’d love to help with it  </p>
+        </div>
```
Comment (mckent05) [inline index.html:207]: **OPTIONAL** - For semantic purposes, I suggest that you remove this section and place it just outside the `form` tag,  but still inside the `section` tag with `class=cardcontact`, please fix this 👍 👍

File: index.html [hunk 3]
```diff
@@ -155,43 +156,70 @@ <h3 class="Multi-Post_05" >Multi-Post Stories </h3>
...
+            <div class="form-group">
+                 <input class="texform" type="text"   id="Name"    classe="form-control" placeholder="Enter Your Name" minlength="4" maxlength="30" required />
+            </div>
+            <div class="form-group">          
+                <input class="texform"  type="email"  id="email"  name="email"   placeholder="Enter Your Email"  required  />
+            </div>
+            <div class="form-group2">
+                <input class="texform2"  name="comment"  id="comment" maxlength="500"s  placeholder="Write your message here..."  required  />
```
Comment (mckent05) [inline index.html]: - According to the project requirements,  please use a `textarea` tag here 🙏 .

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @DamenaM, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Your contact form works as expected 😄 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -155,43 +156,68 @@ <h3 class="Multi-Post_05" >Multi-Post Stories </h3>
...
+        <p class="contact"> If you have an application you are interested 
+            in developing, a feature that you need built or a project that 
+            needs coding. I’d love to help with it  </p>
+        </div>
+            <div class="form-group">
+                 <input class="texform" type="text"   id="Name"    classe="form-control" placeholder="Enter Your Name" minlength="4" maxlength="30" required />
+                <input class="texform"  type="email"  id="email"  name="email"   placeholder="Enter Your Email"  required  />
+            </div >
+            <div class="form-group2">
+                <textarea class="comment"  name="comment-section" id="comment"  maxlength="500"s  placeholder="Write your message here..."  required ></textarea>
```
Comment (huseyinbiyik) [inline index.html:213]: [OPTIONAL] Kindly check the margin and padding, font family on the Figma template and fix them. e.g. padding-top on the text area. You have 45px and a Figma template, 15px. Please stick to the design for all sizes.
 Figma template:
Screenshot 2022-12-15 at 12 05 18 AM

Your design:
Screenshot 2022-12-15 at 12 06 06 AM

**Review by huseyinbiyik — APPROVED** [review]
Hi @DamenaM,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text



## Highlights
- GitHub flow followed 📂
- Linter checks are passing 🟢

## Optional suggestions
- Kindly check the inline comment.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**